ush/ioda/bufr2ioda/bufr2ioda_adpsfc_prepbufr.py

```diff
@@ -107,25 +110,30 @@ def bufr_to_ioda(config, logger):
 
     logger.debug(" ... Executing QuerySet: get ObsType ...")
     # ObsType
+    logger.debug(" ... Executing QuerySet: get ObsType ...")
```

Comment:
isn't this going to print out twice in a row an identical message?

Reply:
True. Missed that. I'll delete it.